install/kubernetes/helm/istio/values-istio-demo.yaml

```diff
@@ -30,3 +30,6 @@ servicegraph:
 
 galley:
   enabled: true
+
+kiali:
+  enabled: true
```

Comment:
We need a kiali automated test case before we can consider this is enabled in the demo profile....

Also, any concern on cpu/mem consumed or stability of kiali to be enabled by default in the demo profile?   cc @costinm @rshriram @sdake @gyliu513

Reply:
cc @jeffmendoza @jmazzitelli

Reply:
linsun, I changed this values to false for the time being.
I will send another PR with testing then I can send another PR enabling it.